assignment2.java

```diff
@@ -1,13 +1,13 @@
 import java.io.InputStream;
 import java.io.FileInputStream;
 import java.io.IOException;
-//import java.util.Scanner;
 import java.util.HashMap;
+import java.util.HashSet;
+import java.util.ArrayList;
 
 public class assignment2{
     public static void main(String[] args)
-    {
-        
+    {   
 
         try (
```

Comment:
I think it'd be a little cleaner to make the try-catch and the inputStream closer to where we use it with the loop starting on line 52. [https://www.w3schools.com/java/java_try_catch.asp](url). Would probably be good to change the parentheses to brackets